QSB/TimeSync/PreventShipDestruction.cs

```diff
@@ -6,14 +6,17 @@
 
 namespace QSB.TimeSync
 {
+    /// <summary>
+    /// Client-only-side component for stopping the ship from being damaged.
+    /// </summary>
     public class PreventShipDestruction : MonoBehaviour
     {
         private void Awake()
         {
-            QSB.Helper.HarmonyHelper.Transpile<ShipDetachableLeg>("Detach", typeof(Patch), "ReturnNull");
-            QSB.Helper.HarmonyHelper.Transpile<ShipDetachableModule>("Detach", typeof(Patch), "ReturnNull");
+            QSB.Helper.HarmonyHelper.Transpile<ShipDetachableLeg>("Detach", typeof(Patch), "ReturnNull"); // Stop legs from detaching
```

Comment:
Another thing with having comments in the same line as the code, is that diffs become harder to read (longer line, and changes in the comment might look like changes in code at a first glance). 

Also means that if one commit makes a change to the comment and another one changes the code, it would count as a conflict.

Version-control-oriented programming 😬